default-plugins/compact-bar/src/line.rs

```diff
@@ -199,7 +199,7 @@ fn tab_line_prefix(
         tab_index: None,
     }];
     if let Some(name) = session_name {
-        let name_part = format!("({}) ", name);
+        let name_part = format!("({})", name);
```

Comment:
This is the session-name part, remove the whitespace on the right, the mode part will handle padding

![image](https://github.com/zellij-org/zellij/assets/6853656/258cec04-90bd-4d34-ab79-1e21a78ece6e)